EDU 266: How to schedule a Workflow in Go (#2005)
temporalio/documentation: docs-src/go/how-to-describe-a-schedule-in-go.md

---
id: how-to-describe-a-schedule-in-go
title: How to describe a Schedule in Go
sidebar_label: Describe Schedule
description: Describe a Schedule in Go.
---

`Describe` retrieves information about the current Schedule configuration.

> This can include details about the Schedule Spec (such as Intervals), CronExpressions, and Schedule State.

Author: I can put it in the parentheses, then

> This can include details about the Schedule Spec (such as Intervals), CronExpressions, and Schedule State.

Author: ```suggestion
This can include details about the Schedule Spec (such as Intervals or, CronExpressions)
```suggestion
This can include details about the Schedule Spec (such as Intervals), CronExpressions, and Schedule State.
``` and Schedule State.
```
